crates/collab/README.md

```diff
@@ -8,7 +8,12 @@ It contains our back-end logic for collaboration, to which we connect from the Z
 
 ## Database setup
 
-Before you can run the collab server locally, you'll need to set up a zed Postgres database.
+Before you can run the collab server locally, you'll need to set up a zed Postgres database. Follow the steps sequentially:
+
+1. Ensure you have postgres installed. If not, install with `brew install postgresql@15`.
```

Comment:
I'm aware the instructions are Mac-oriented. Happy to frame the instructions in a more platform-agnostic way if there's push back.

Reply:
Always gotta to leave something for the next person.